File: app/src/main/java/closer/vlllage/com/closer/handler/helpers/TopHandler.java
package closer.vlllage.com.closer.handler.helpers;

import closer.vlllage.com.closer.pool.PoolMember;

public class TopHandler extends PoolMember {

    private String activeGroupId = null;

    public boolean isGroupActive(String groupId) {
        return activeGroupId != null && activeGroupId.equals(groupId);
    }

    public void setGroupActive(String groupId) {
        activeGroupId = groupId;
    }
}

File: app/src/main/java/closer/vlllage/com/closer/handler/helpers/TopHandler.kt
package closer.vlllage.com.closer.handler.helpers

import com.queatz.on.On

class TopHandler constructor(private val on: On) {

    private var activeGroupId: String? = null

    fun isGroupActive(groupId: String): Boolean {
        return activeGroupId != null && activeGroupId == groupId
    }

    fun setGroupActive(groupId: String?) {
        activeGroupId = groupId
    }
}
